accord-core/src/test/java/accord/burn/BurnTestConfigurationService.java

```diff
@@ -66,7 +70,7 @@ public FetchTopologyRequest(long epoch)
         public void process(Node on, Node.Id from, ReplyContext replyContext)
         {
             Topology topology = on.configService().getTopologyForEpoch(epoch);
```

Comment:
this can throw, so prob want a `try/catch`?

Reply:
posted this in slack, prob want to change `Node.recieve` to have


```
scheduler.now(() -> {
            try
            {
                request.process(this, from, replyContext);
            }
            catch (Throwable t)
            {
                reply(from, replyContext, null, t);
            }
        });
```

that way you don't worry about my above comment